apps/etterlatte-behandling/src/main/kotlin/behandling/etteroppgjoer/forbehandling/EtteroppgjoerForbehandlingDao.kt

```diff
@@ -41,6 +41,25 @@ class EtteroppgjoerForbehandlingDao(
 ) {
     private val logger: Logger = LoggerFactory.getLogger(EtteroppgjoerForbehandlingDao::class.java)
 
+    fun hentForbehandlingMedSvarfristUtloept(inntektsAar: Int): List<EtteroppgjoerForbehandling>? =
+        connectionAutoclosing.hentConnection {
+            with(it) {
+                val statement =
+                    prepareStatement(
+                        """
+                        SELECT *  
+                        FROM etteroppgjoer_behandling
+                        WHERE varselbrev_sendt IS NOT NULL
+                        AND varselbrev_sendt < (now() - interval '1 month')
```

Comment:
Jeg tenker mengden tid (1 måned) kanskje bør være et parameter vi konfigurerer opp for jobben? Det virker rimelig for meg at vi vil f.eks. kjøre med mye kortere "frist" i dev for å teste denne funksjonaliteten